__tests__/app.test.js

```diff
@@ -94,6 +94,47 @@ describe("GET /api/reviews", () => {
   });
 });
 
+describe.only("GET /api/reviews/:review_id/comments", () => {
```

Comment:
please don't commit `.only` in your tests ... kinda defeats the point of a test suite, right